ui/operator_ui/templates/index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
+    <base href="{{app_url}}">
```

Comment:
@simo9000 unfortunately, this change introduces a breaking change, see #2302. Now everybody has to set appUrl because with the default localhost appUrl as base the app will not load. App URL is only used for authorization. The rest is full links. Maybe most users haven't used it so having it hardcoded in deployment.yaml template was fine. I have to read up on ingress rules, but can you tell us why you need this to be changed?